pkg/helm/charts/tests/securedclusterservices/testdata/helmtest/injected-cabundle-cm.test.yaml

```diff
@@ -13,15 +13,6 @@ tests:
   - name: "Injected CA bundle has a proper label"
     expect: |
       assertThat(.configmaps["injected-cabundle-stackrox-secured-cluster-services"].metadata.labels["config.openshift.io/inject-trusted-cabundle"] == "true")
-- server:
-    visibleSchemas:
-    - openshift-3.11.0
-  set:
-    env.openshift: 3
-  tests:
-  - name: "No injected CA bundle on Openshift 3"
-    expect: |
-      assertThat(.configmaps["injected-cabundle-stackrox-secured-cluster-services"] == null)
```

Comment:
I think, this should be kept. Why would we want to remove that?

Reply:
Done!